content/it/docs/reference/glossary/etcd.md

```diff
@@ -0,0 +1,21 @@
+---
+title: etcd
+id: etcd
+date: 2018-04-12
+full_link: /docs/tasks/administer-cluster/configure-upgrade-etcd/
+short_description: >
+  È un database key-value altamente disponibile che è usato da Kubernetes per salvare tutte le informazioni del cluster.
+
+aka: 
+tags:
+- architecture
+- storage
+---
+ È un database key-value altamente disponibile che è usato da Kubernetes per salvare tutte le informazioni del cluster.
```

Comment:
"altamente disponibile" does not make sense to me in Italian. I suggest to remove it